django/db/models/sql/compiler.py

```diff
@@ -290,11 +282,10 @@ def get_order_by(self):
         else:
             ordering = []
         if self.query.standard_ordering:
-            asc, desc = ORDER_DIR['ASC']
+            asc, _desc = ORDER_DIR['ASC']
         else:
-            asc, desc = ORDER_DIR['DESC']
+            asc, _desc = ORDER_DIR['DESC']
```

Comment:
`asc` is misleading, because it's basically a default ordering. Renamed to the `default_order`.